packages/cosmic-swingset/lib/ag-solo/vats/vat-wallet.js

```diff
@@ -95,10 +95,16 @@ function build(E, _D, _log) {
         };
       }
       case 'walletAcceptOffer': {
-        const ret = await wallet.acceptOffer(data);
+        const { outcome } = await wallet.acceptOffer(data);
+        let result = { outcome: 'Offer was made.' };
```

Comment:
```suggestion
        // We return the outcome only if it is a string. Otherwise we
        // return a default message.
        const result = (typeof outcome === 'string') 
            ? { outcome }
            : { outcome: 'Offer was made.' };
```

Reply:
Looks better, thanks